.buildkite/commands/run-ui-tests.sh

```diff
@@ -18,6 +18,12 @@ sudo ln -sfn /usr/local/opt/openjdk@11/libexec/openjdk.jdk /Library/Java/JavaVir
 echo "--- :rubygems: Fixing Ruby Setup"
 gem install bundler
 
+echo "--- :test-analytics: Leave note about missing analytics for UI tests"
+buildkite-agent annotate \
+  'Test Analytics for UI tests are currently unavailable' \
```

Comment:
Should we add a link to a GitHub issue tracking this?

Especially to explain that this is a _configuration_ issue on our end and you're investigating why — because I'm now thinking that the wording could let people think that "oh ok Buildkite doesn't support test analytics for UITests" or "oh something went wrong with Test Analytics on this build, maybe an outage" or even "What are Test Analytics, does that mean my junit/test results were not produced?" for people not familiar with Buildkite's Test Analytics feature, rather than reading it as "the Owl team is working on it"

Reply:
Alternatively to having a GitHub issue with more info, at least we could change the wording to this:

```suggestion
  "Buildkite Test Analytics are currently not reported for UITests. We're investigating why and working on fixing it." \
```